sympy/__init__.py

```diff
@@ -10,6 +10,15 @@
 """
 
 from __future__ import absolute_import, print_function
+del absolute_import, print_function
+
+try:
+    import mpmath
+except ImportError:
+    raise ImportError("SymPy now depends on mpmath as an external library. "
+    "You can install mpmath with 'pip install mpmath'.")
```

Comment:
Maybe "see http://docs.sympy.org/latest/install.html for more information" (except that url won't be correct until we release).